sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -478,14 +483,16 @@ def add_participant(
             thread_participant,  # type: ChatThreadParticipant
             **kwargs  # type: Any
     ):
-        # type: (...) -> None
+        # type: (...) -> tuple(ChatThreadParticipant, CommunicationError)
```

Comment:
@annatisch we mentioned this one on the .NET review:
https://apiview.dev/Assemblies/Review/a60a130cc1e24d65a74c20e42887b61f#74d339479f13486ebababaee478f8990

Reply:
@annatisch `add_participant` raises error - change implemented

Reply:
@annatisch are we ok with having this differ from the other sdks ? I am ok if you are. 
When we run this in that meeting, the answer was to leave it as is in case in the future any additional data is required back from that call. I don't see that coming but that was the decision.

Reply:
@juancamilor - raising errors rather than returning them is Pythonic behaviour that I would expect to differ from .Net.
We can still return the `ChatThreadParticipant` object - when you say additional data, you mean other information outside of these two fields?

Reply:
yes, something like extra headers is what  Ted G mentioned, but I don't have any requirements yet to back it up.

tg-msft1
There aren't today, but maybe it'll come back with extra headers in the future?

Reply:
@annatisch after today's review we will address all the changes on the next iteration (including the removal of this operation).